### PR TITLE
Remove remnants of Python from package.json

### PR DESCRIPTION
**Pull request by azz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,13 @@
 {
-  "name": "@prettier/python",
+  "name": "prettier-plugin-python",
```
Comment (j-f1) [inline package.json]: `prettier-plugin-java`?

Comment (azz) [reply package.json]: d'oh!

**Comment by thorbenvh8** [comment]
@j-f1 thanks for finding the bug :)
@azz thanks for the pr :)

**Comment by azz** [comment]
It's 4am. I need sleep. 💤

**Pull request merged**